units/src/amount/mod.rs

```diff
@@ -582,8 +582,8 @@ enum DisplayStyle {
 
 /// Calculates the sum over the iterator using checked arithmetic.
 pub trait CheckedSum<R>: sealed::Sealed<R> {
-    /// Calculates the sum over the iterator using checked arithmetic. If an over or underflow would
-    /// happen it returns [`None`].
+    /// Calculates the sum over the iterator using checked arithmetic. If an
+    /// overflow would happen it returns [`None`].
```

Comment:
happens

Reply:
Either is correct I think, but I can change it to what you suggested.

Reply:
Ok, I updated it to say `if an overflow happens..`.  For future reference though, why do you prefer that over `If an overflow would happen..`

Reply:
Crap, sorry that was a brainfart. I think my brain didn't register "would" for some reason.

But still it does seem more natural to have happens - possibly even demonstrated by that brainfart.